### PR TITLE
[ Pooling2D ] Pooling 2D Layer : Draft

### PR DESCRIPTION
**Pull request by jijoongmoon**
This PR provides skeleton of pooling 2d layer

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: jijoong.moon <jijoong.moon@samsung.com>

### CODE REVIEW

File: nntrainer/src/parse_util.cpp [hunk 1]
```diff
@@ -96,10 +96,11 @@ unsigned int parseType(std::string ll, InputType t) {
    *            "fully_conntected" : Fully Connected Layer Object
    *            "batch_normalization" : Batch Normalization Layer Object
    *            "conv2d" : Convolution 2D Layer Object
+   *            "pooling2d" : Pooling 2D Layer Object
    *            "unknown" : Batch Normalization Layer Object
```
Comment (dongju-chae) [inline nntrainer/src/parse_util.cpp:100]: BTW, it seems like a mistake.

File: nntrainer/src/parse_util.cpp [hunk 2]
```diff
@@ -233,17 +252,20 @@ unsigned int parseLayerProperty(std::string property) {
...
+   * Pooling2DLayer has 12, 13, 14, 15 properties.
    * BatchNormalizationLayer has 0, 1, 5, 6, 7 properties.
    */
-  std::array<std::string, 15> property_string = {
+  std::array<std::string, 17> property_string = {
     "input_shape", "bias_zero",  "normalization", "standardization",
     "activation",  "epsilon",    "weight_decay",  "weight_decay_lambda",
     "unit",        "weight_ini", "filter",        "kernel_size",
-    "stride",      "padding",    "unknown"};
+    "stride",      "padding", "pooling_size", "pooling", "unknown"};
```
Comment (dongju-chae) [inline nntrainer/src/parse_util.cpp:268]: it seems to need some arrangement fix

Comment (jijoongmoon) [reply nntrainer/src/parse_util.cpp]: fixed. Thanks.!

File: nntrainer/src/parse_util.cpp (hunk 1 above)
Comment (jijoongmoon) [reply nntrainer/src/parse_util.cpp]: fixed. Thanks.!

**Pull request merged**